storage/innobase/fsp/fsp0file.cc

```diff
@@ -531,9 +538,9 @@ Datafile::validate_first_page(lsn_t* flush_lsn)
 
 	if (error_txt != NULL) {
 err_exit:
-		ib::info() << error_txt << " in datafile: " << m_filepath
-			<< ", Space ID:" << m_space_id  << ", Flags: "
-			<< m_flags;
+		sql_print_error("InnoDB: %s in datafile: %s, Space ID: %d, "
+				"Flags: %d", error_txt, m_filepath,
+				m_space_id, m_flags);
```

Comment:
`m_space_id` and `m_flags` are not `int`. They require `ULINTPF` or `"%zu"` in this version.